var/spack/repos/builtin/packages/atop/package.py

```diff
@@ -28,10 +28,13 @@
 class Atop(Package):
     """Atop is an ASCII full-screen performance monitor for Linux"""
     homepage = "http://www.atoptool.nl/index.php"
-    url      = "http://www.atoptool.nl/download/atop-2.2-3.tar.gz"
+    url = "http://www.atoptool.nl/download/atop-2.2-3.tar.gz"
```

Comment:
Just so you know, all lines in `package.py` that start with `url` or contain `http` are ignored from the flake8 line length checks. So there's no need to change these url lines.